.github/dependabot.yml

```diff
@@ -10,3 +10,9 @@ updates:
       # For all packages, ignore all major versions to minimize breaking issues
       - dependency-name: "*"
         update-types: ["version-update:semver-major"]
+  - package-ecosystem: "github-actions"
```

Comment:
Why not include `version-update:semver-major` like was recently added onto gradle dependencies?

We have fewer (I think?) dependencies in GitHub Actions and I'd like to try it before I see lots of churn.  Easy to make this change as a fast follow if there is concern.

Reply:
I think for Github actions the update to major is not as breaking as for main codebase, most do major anyway (fe codeql v1 -> v2, checkout v2 -> v3, ...)